src/main/java/CalculatorInputText.java

```diff
@@ -0,0 +1,18 @@
+public class CalculatorInputText {
+
+    private final String str;
+    private StringSplitPolicy stringSplitPolicy;
+
+    public CalculatorInputText(String str) {
+        this.str = str;
+        this.stringSplitPolicy = new StringSplitPolicy(str);
```

Comment:
입력 값이 들어오자마자 StringSplitPolicy를 통해 split하여 strs에 할당하는 건 어떠신가요?
```suggestion
public class CalculatorInputText {
    private final String[] strs;
    ...
    public CalculatorInputText(String str) {
        this.str = StringSplitPolicy.splitByPolicy(str)로 
    }
```